.gitignore

```diff
@@ -55,3 +58,10 @@ webpack/node_modules
 
 # Assets
 assets/js/
+
+# indentfix
+packages/indentfix/dist
```

Comment:
wenn wir das in die globale gitignore übernehmen wollen macht es vermutlich Sinn das etwas generischer als:
```
packages/*/dist
packages/*/build
packages/*/*.pyc
packages/*/*.pyo
packages/*/*.egg-info
```
so dass es auch gleich entsprechend für andere Plugins anwendbar ist, sollten da jemand welche ebenfalls auf diesem Wege hinzugefügt werden.

Reply:
done